modules/tinymce/src/plugins/image/main/ts/ui/MainTab.ts

```diff
@@ -79,8 +81,7 @@ const makeItems = (info: ImageDialogInfo) => {
     info.hasImageTitle ? [ imageTitle ] : [],
     info.hasDimensions ? [ imageDimensions ] : [],
     [{
-      type: 'grid',
-      columns: 2,
+      ...dialogType(info.hasImageCaption),
```

Comment:
This isn't quite correct sorry, as it needs to fallback to a panel if either the class list or the caption is disabled. At the moment it'll only do that if the caption is disabled.

To word it differently it should only be using the grid layout with 2 columns when both the classes and caption need to be shown.

Reply:
updated